packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -216,7 +219,11 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
     private doHandleMarkerFilterChangedSignal(annotationMarkers: string[]) {
         this.annotationMarkers = annotationMarkers;
```

Comment:
It would help for clarity if everywhere that annotation/marker category is used (parameters, variables) to use 'category' in the name. Also "annotation markers" is redundant, they are synonyms.